src/features.ts

```diff
@@ -12,7 +12,7 @@ export const features = {
   startWithIndexScreen: __DEV__ ? false : false,
   prefillWalletInfo: __DEV__ ? false : false,
   showProdPoolsInDev: __DEV__ ? false : false,
-  showNftGallery: __DEV__ ? true : false,
+  showNftGallery: __DEV__ ? true : true,
```

Comment:
nft gallery uses the same metadata endpoint thats not currently usable

Reply:
it's scrapping it correctly